### ISSUE TITLE
Support linestring in merge analyser

### ISSUE THREAD

**Issue by frodrigo** [osm-fr/osmose-backend#768]
Merge analyser only work with nodes. Add support of linestrings.

**Comment by frodrigo** [comment]
@didier2020 convert linestring to point on import then conflate only with point is easy. Other part require more works.

**Comment by didier2020** [comment]
i agree but if the source to merge is linestring, conflation is on one or more way ?
(a point is not enough to identify the beginning and the end)

**Comment by frodrigo** [comment]
We have to review all the https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py
But it is one of the most complex part of the Backend.

Main changes:
- [ ] Import line (or poly)
- [ ] conflate geometry, choose a way to compare

**Comment by didier2020** [comment]
ok, so i just fix linestring for geojson.

**Comment by frodrigo** [comment]
Already done
